bats/tests/helpers/vm.bash

```diff
@@ -1,3 +1,7 @@
+local_setup_file() {
```

Comment:
This looks wrong; `local_*` functions should only be defined in actual `*.bats` test files, and not in the helpers.

If this should be applied globally, then it should be in `setup_file` in `load.bash`.

Reply:
Thanks, will fix:
- I'll add `bats_require_minimum_version 1.5.0` to `wait_for_extension_manager()` (because that's the minimum that the function needs)
- Also add `bats_require_minimum_version 1.10.0` to `setup_file()` in `load.bash` (since that's the version of our submodule, which is the only version we test against).

Reply:
I think it would be good to add a comment on why we need a specific version.

I also think we don't need to add tests for specific versions that are lower than what we will be checking for in `load.bash`. They could never fail, since we always source `load.bash` first.

Reply:
I disagree; that is just clutter. You declare the minimum version once at the top of the program, and then you can use any feature of the library included in that version or earlier.

The same way we only check for `$BASH_VERSION` once, instead sprinkling little checks all over the place: this function requires bash 4.2 because..., and this function need s bash 5.x because...

Reply:
Alright, made it one centralized call instead